se-test/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+This task is intended for people who interviews our software engineer roles:
+ - [Software Engineer](https://condenastuk.avature.net/careers/JobDetail/Software-Engineer/1667)
+ - [Senior Software Engineer](https://condenastuk.avature.net/careers/JobDetail/Senior-Software-Engineer/1412) 
+ - [Technical Lead Engineer](https://condenastuk.avature.net/careers/JobDetail/Technical-Lead-Software-Engineer/1468) 
+
+Please make sure you read [our expectations](../README.md#What we are looking for?) first before getting down to the following tasks:  
+
+## Option 1
+
+Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see the latest **UK news** and give the ability to the user to filter them by keywords (see [everything endpoint api](https://newsapi.org/docs/endpoints/everything)). Optionally clicking on a news the user will navigate to an individual article page.
+
+Given the fact that this is a Full Stack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
+
+A Full Stack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
+
+### Submission
+See ["how to submit your work"](../README.md#how-to-submit-code)
+
+### FAQ
+
+Q: Shall I provide my `newsapi.org` token?  
+A: Absolutely not. Provide a way we can easily plug it in
```

Comment:
```suggestion
A: No, but please provide one for us
```
`Absolutely not` sounds quite aggressive. As above, I know this isn't your intent.